app/controllers/instructeurs/dossiers_controller.rb

```diff
@@ -498,5 +487,26 @@ def redirect_on_dossier_in_batch_operation
         redirect_back(fallback_location: instructeur_dossier_path(procedure, dossier_in_batch))
       end
     end
+
+    def set_gallery_attachments
+      gallery_attachments_ids = Rails.cache.fetch([dossier, "gallery_attachments"], expires_in: 10.minutes) do
+        champs_attachments_ids = dossier
+          .champs
+          .filter { _1.class.in?([Champs::PieceJustificativeChamp, Champs::TitreIdentiteChamp]) }
```

Comment:
```suggestion
          .where(type: [Champs::PieceJustificativeChamp.name, Champs::TitreIdentiteChamp.name])
```
à moins que le filter soit volontaire ?

Reply:
bien vu 👍 c'était pas intentionnel, j'ai modifié